examples/cstubs_structs/myocamlbuild.ml

```diff
@@ -0,0 +1,35 @@
+open Ocamlbuild_plugin
+
+let () =
+  let additional_rules = function
+    | Before_hygiene  -> ()
+    | After_hygiene   -> ()
+    | Before_options  -> ()
+    | After_options   -> ()
+    | Before_rules    -> ()
+    | After_rules     ->
+
+      (* Generate stubs. Steps 1, 2, & 3 of Makefile (1 & 2 via built-in rules).
+        ML -> C *)
+      rule "cstubs: x_c_gen.native -> x_stubs_gen.c"
+        ~dep:"%_c_gen.native"
+        ~prod:"%_stubs_gen.c"
+        (fun env _build -> Cmd (A (env "./%_c_gen.native")));
+                 (* OCAMLBULD_BUG: the './' is necessary *)
```

Comment:
Has the bug been reported?  It would be helpful to have a link to the report here, so we know when we can safely remove the comment and workaround.